### PR TITLE
Add config support for SDK transport security

### LINKED ISSUE #727: Add config support for SDK transport security
### Problem

The SDK support `setTransportSecurity()` on the client

### Solution

Add a configuration option and set the Transport security option appropriately.
This allows support for TLS 

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by natanasow**
Signed-off-by: nikolay <n.atanasow94@gmail.com>

**Description**:
Update relay.ts to set transport based on value of new CLIENT_TRANSPORT_SECURITY config

**Related issue(s)**:

Fixes #727

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looking good
Please update PR comments with something like
`Update relay.ts to set transport based on value of new CLIENT_TRANSPORT_SECURITY config`

Additionally, you need to expose this in the Helm charts `values.yaml` and `deployment.yaml` so it can be utilized

File: helm-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -173,6 +173,12 @@ spec:
                 name: {{ include "json-rpc-relay.fullname" . }}
                 key: MIRROR_NODE_LIMIT_PARAM
                 optional: true
+          - name: CLIENT_TRANSPORT_SECURITY
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "json-rpc-relay.fullname" . }}
+                key: CLIENT_TRANSPORT_SECURITY
+                optional: false
```
Comment (Nana-EC) [inline helm-chart/templates/deployment.yaml:181]:
```suggestion
                optional: true
```

Comment (georgi-l95) [reply helm-chart/templates/deployment.yaml]: Done.

**Review by Nana-EC — APPROVED** [review]
LG

**Pull request merged**